website/docs/contributing/writing-guide.md

```diff
@@ -5,11 +5,34 @@ draft: true
 
 # Writing guide
 
+## Front matter
+
+The front matter represents the metadata for each page.
+It is written at the top of each page and must be enclosed by `---` on both sides.
+Example:
+
+```yaml
+---
+sidebar_position: 1
+slug: /writing-guide
+description: How to write documentation
+---
+```
+
 ## Names and URLs
 
 Use `kebab-case-with-dashes` instead of `snake_case_with_underscores` or spaces
 for file names, directory names, and slugs because URL paths typically use dashes.
 
+Ensure that the file name/URL path matches the title of the page.
+For example, if the title of your page is "Getting Started", then the file name/URL path (`slug` in front matter) should also be "getting-started" to maintain consistency.
```

Comment:
I think we should recommend omitting `slug:` be default, and say that it only for special cases like external links, backward compatibility, etc